backend/src/plugins/kube.ts

```diff
@@ -58,7 +64,9 @@ export default fp(async (fastify: FastifyInstance) => {
       .readNamespacedConfigMap('console-config', 'openshift-console')
       .then((result) => result.body);
     if (consoleConfig.data?.[CONSOLE_CONFIG_YAML_FIELD]) {
-      const consoleConfigData = jsYaml.load(consoleConfig.data[CONSOLE_CONFIG_YAML_FIELD]);
+      const consoleConfigData = jsYaml.load(
+        consoleConfig.data[CONSOLE_CONFIG_YAML_FIELD],
+      ) as ConsoleConfigData;
```

Comment:
Inline the type definitions here
```suggestion
      const consoleConfigData = jsYaml.load(
        consoleConfig.data[CONSOLE_CONFIG_YAML_FIELD],
      ) as { customization?: {  branding?: string; }; };
```